ert3_examples/spe1/experiments/evaluation/ensemble.yml

```diff
@@ -13,13 +13,7 @@ input:
 
 output:
     -
-      record: WOPT:PROD
-    -
-      record: WWPT:PROD
-    -
-      record: WGPT:PROD
-    -
-      record: WWIT:INJ
+      record: eclsum
```

Comment:
I think this warrants some documentation in the `README.md` of this `spe1` example.
Readability is slightly decreased and consistency with the `sensitivity/ensemble.yml` that still has
```
    -
      record: WOPT:PROD
    -
      record: WWPT:PROD
    -
      record: WGPT:PROD
    -
      record: WWIT:INJ
```
The `spe1` example should showcase ert3 functionality, which this discrepancy actually does.
It only needs (in my opinion) to be documented explicitly under "Evaluation" in the "The experiments" section.
Notice that there is precedence for this, as `stochastic.field_properties` is already explained here.